openedx/core/djangoapps/credentials/settings/aws.py

```diff
@@ -0,0 +1,6 @@
+"""Production settings for Credentials"""
+
+
+# def plugin_settings(settings):
```

Comment:
are there settings for this?  If so we should remove them from aws.py and add them here.